### PR TITLE
Addition of ganglia package

### PR DESCRIPTION
**Pull request by navjotsinghji**
###### Merge Checklist  

**All** boxes should be checked before merging the PR *(just tick any boxes which don't apply to this PR)*
- [x] The toolchain has been rebuilt successfully (or no changes were made to it)
- [x] The toolchain/worker package manifests are up-to-date
- [x] Any updated packages successfully build (or no packages were changed)
- [ ] Packages depending on static components modified in this PR (Golang, `*-static` subpackages, etc.) have had their `Release` tag incremented.
- [ ] Package tests (%check section) have been verified with RUN_CHECK=y for existing SPEC files, or added to new SPEC files
- [ ] All package sources are available
- [x] cgmanifest files are up-to-date and sorted (`./cgmanifest.json`, `./toolkit/scripts/toolchain/cgmanifest.json`, `.github/workflows/cgmanifest.json`)
- [x] LICENSE-MAP files are up-to-date (`./SPECS/LICENSES-AND-NOTICES/data/licenses.json`, `./SPECS/LICENSES-AND-NOTICES/LICENSES-MAP.md`, `./SPECS/LICENSES-AND-NOTICES/LICENSE-EXCEPTIONS.PHOTON`)
- [x] All source files have up-to-date hashes in the `*.signatures.json` files
- [ ] `sudo make go-tidy-all` and `sudo make go-test-coverage` pass
- [ ] Documentation has been updated to match any changes to the build system
- [x] Ready to merge

---

###### Summary 

What does the PR accomplish, why was it needed?

###### Change Log  

- Change
- Change
- Change

###### Does this affect the toolchain?  

**YES/NO**

###### Associated issues  

- #xxxx

###### Links to CVEs  
- https://nvd.nist.gov/vuln/detail/CVE-YYYY-XXXX

###### Test Methodology

- Pipeline build id: xxxx

### CODE REVIEW

File: SPECS-EXTENDED/ganglia/ganglia.signatures.json [hunk 1]
```diff
@@ -0,0 +1,11 @@
+{
+ "Signatures": {
+  "ganglia-web-3.7.5.tar.gz": "067f791287a11210d274acf4c9c1b5e3df3482ab3f3a151071cf6c926460cfa9",
```
Comment (mfrw) [inline SPECS-EXTENDED/ganglia/ganglia.signatures.json:3]: I do not think the sources are uploaded to the blobstore.
I can upload them on your behalf.

File: SPECS-EXTENDED/ganglia/ganglia.spec [hunk 2]
```diff
@@ -0,0 +1,763 @@
+%global gangver     3.7.2
+%global webver      3.7.5
+
+%global systemd         1
+%global _hardened_build 1
+
+%if 0%{?rhel} && 0%{?rhel} < 8
+%global py2             1
```
Comment (mfrw) [inline SPECS-EXTENDED/ganglia/ganglia.spec:8]: We so not support python2; only python3

File: SPECS-EXTENDED/ganglia/ganglia.signatures.json (hunk 1 above)
Comment (navjotsinghji) [reply SPECS-EXTENDED/ganglia/ganglia.signatures.json:3]: yes..Thanks

**Comment by navjotsinghji** [comment]
I am going to submit this request for SPECS, so abandoning this PR.

**Pull request closed without merging**